web/app.yaml

```diff
@@ -12,3 +12,8 @@ handlers:
   - url: /.*
     static_files: build/index.html
     upload: build/index.html
+
+env_variables:
+  # Replace with API key and authentication domain before deployment
```

Comment:
Can we create a readme file as deployment playbook? (future PR)

Reply:
Okay! Created issue #69.